File: Express JS/projectExpress/app/models/Note.ts
export class Note{
    private id: number = -1;
    private content: string = "";
    private userId: number = -1;
    //Constructor
    constructor(
        id: number,
        content: string,
        userId: number
    ){
        this.id = id;
        this.content = content;
        this.userId = userId;
    }
    //Getters
    get Id():number{
        return this.id;
    }
    get Content():string{
        return this.content;
    }
    get UserId():number{
        return this.userId;
    }
    //Setters
    set Id(id:number){
        this.id = id;
    }
    set Content(content:string){
        this.content = content;
    }
    set UserId(userId:number){
        this.userId = userId;
    }
}

File: Spring Boot/project/src/main/java/com/milestone/project/models/Note.java
package com.milestone.project.models;

import com.google.gson.JsonObject;

import javax.persistence.*;
//JPA ENTITY
@Entity
@Table(name = "notes")
public class Note {
    @Id
    @GeneratedValue(
            strategy = GenerationType.IDENTITY
    )
    private Long id;
    @Column(name = "content", nullable = false)
    private String content;
    @Column(name = "user_id", nullable = false)
    private Long userId;
    //Default Constructor
    public Note(){}
    //Constructor
    public Note(
            String content,
            Long userId
    ){
        this.content = content;
        this.userId = userId;
    }
    //Getters
    public Long getId() {
        return id;
    }

    public String getContent() {
        return content;
    }

    public Long getUserId() {
        return userId;
    }
    //Setters
    public void setContent(String content) {
        this.content = content;
    }
    //toString with JSON
    @Override
    public String toString() {
        JsonObject json = new JsonObject();
        json.addProperty("id", this.getId());
        json.addProperty("content", this.getContent());
        json.addProperty("userId", this.getUserId());
        return json.toString();
    }
}
